distributions/Dist_ams1396.py

```diff
@@ -10,15 +10,15 @@ def __init__(self):
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return (5*x^11)
+		return (5*(x**11))
 
 	def mean(self):
 		"""This is the mean of the PDF"""
 		return 0.
 
 	def std(self):
 		"""This is the standard deviation of the pdf"""
-		return (.000244140625) 
+		return ((5/14)*(((dist.x_max)**14) - ((dist.x_min)**14))) 
```

Comment:
glad to see equation , but you need 5./14 because in most computer programming languages 5/14 uses integer division and gives you 0.